src/components/CategoryFeatureLink/CategoryFeatureLink.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link, View } from '@aws-amplify/ui-react';
+
+interface CategoryFeatureLinkProps {
+  isExternal: boolean;
+  href: string;
+  children: React.ReactNode;
+}
+
+const CategoryFeatureLink: React.FC<CategoryFeatureLinkProps> = ({
+  href,
+  isExternal,
+  children
+}) => {
+  return (
+    <View as="li">
+      <Link
+        href={href}
+        className="external-link"
+        isExternal={isExternal}
+        textDecoration={'underline'}
+      >
+        {children}
+      </Link>
```

Comment:
You don't have to change this now, but we might have to change this to be the `Link` component from `next/link` later so that it can navigate to the pages in the docs app.

I saw below we're hardcoding the platform as "javascript", but what happens if we're on a different homepage for a different platform? How do we build out the correct links from the platform that gets passed into the `PlatformFreatures` component?

Reply:
Since it is a `prop` we can pass the` platform` to the component, javascript should be default, will change thanks.
Good, catch will discuss about the `Link` in offline, thanks